action.yml

```diff
@@ -9,6 +9,10 @@ inputs:
     description: 'GitHub token or PAT token'
     required: true
     default: ${{ github.token }}
+  github_url:
```

Comment:
I'll recommend using `${{ github.server_url }}` to fix this

Reply:
done